ErrorReporting/src/Bootstrap.php

```diff
@@ -232,4 +242,22 @@ public static function shutdownHandler()
             }
         }
     }
+
+    public static function getFunctionNameForReport($trace = [])
```

Comment:
Just a few small nits:

- This will need a description (it is why CI is failing)
- Could use an array type hint
- Should we add a test?

Reply:
Also, I'm wondering if we need this to be public or if it would be safe to minimize the surface and have it be private?

Reply:
@dwsupplee totally good with making it private. I made it public so it was more easily testable and then didn't write any tests for it :P